cli/cmd/resources/datacollection.go

```diff
@@ -34,51 +34,33 @@ func NewDataCollectionClusterRole(psp bool) *rbacv1.ClusterRole {
 			Name: "odigos-data-collection",
 		},
 		Rules: []rbacv1.PolicyRule{
-			{
-				Verbs: []string{
-					"get",
-					"list",
-					"watch",
-				},
+			{ // TODO: remove this after we remove honeycomb custom exporter config
+				// located at: autoscaler/controllers/datacollection/custom/honeycomb.go
 				APIGroups: []string{""},
-				Resources: []string{
-					"pods",
-					"endpoints",
```

Comment:
@tamirdavid1 is it related to the load balancer of the tail sampler maybe?

Reply:
Yea seems like indeed the load balancer uses the endpoints
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/loadbalancingexporter/resolver_k8s.go#L105

also it written in this readme.md
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/loadbalancingexporter/example/k8s-resolver/README.md\

@BenElferink @blumamir

Reply:
No problem, I'll add it again :)